src/main/java/org/folio/rest/repository/OverdueFinePolicyRepository.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.rest.repository;
+
+import java.util.Map;
+
+import org.folio.rest.jaxrs.model.OverdueFinePolicy;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.utils.TenantTool;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+
+public class OverdueFinePolicyRepository {
```

Comment:
This looks a lot like `LostItemFeePolicyRepository` above. Should they extend the same parent? Like we did with `BaseRepository` in mod-patron-blocks maybe?